### PR TITLE
[EK-49] Organize `ros_utilities` for release

### PR DESCRIPTION
**Pull request by mhidalgo-bdai**
This patch re-organizes this repository in preparation for its release to ROS 2. In particular, it:

- Removes `proto2ros*` packages, as these will now live in their own repository.
- Renames `bdai_ros2_wrappers` to ~`rainbows`~ `synchros2` to suit upcoming changes.
   - Yet `bdai_ros2_wrappers` is kept as an alias to ~`rainbows`~ `synchros2`.
- Polishes containerization and packaging.

### CODE REVIEW

**Review by khughes-bdai — APPROVED** [review]
LGTM!
minor question -- is it worth mentioning somewhere in the readme that the proto2ros stuff used to live here, and has now moved to another repo?

**Comment by mhidalgo-bdai** [comment]
> is it worth mentioning somewhere in the readme that the proto2ros stuff used to live here, and has now moved to another repo?

Good idea. Adding it :gear:

**Comment by khughes-bdai** [comment]
also looks like we just got the `opensource@theaiinstitute.com` alias -- can we update that here too?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+-->
+
+### Checklist
+
+<!-- Mark each checkbox as you make progress in your contribution. -->
+
+- [ ] Lint and unit tests pass locally
+- [ ] I have added tests that prove my changes are effective
+- [ ] I have added necessary documentation to communicate the changes
+
```
Comment (amessing-bdai) [inline .github/PULL_REQUEST_TEMPLATE.md:16]: ros-based tests use the domain coordinator

**Comment by mhidalgo-bdai** [comment]
Holding this up until we've sorted out the package rename.

File: bdai_ros2_wrappers/package.xml [hunk 2]
```diff
@@ -2,17 +2,12 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>bdai_ros2_wrappers</name>
-  <version>0.0.1</version>
-  <description>The AI Institute's wrappers for ROS2</description>
-  <maintainer email="engineering@theaiinstitute.com">The AI Institute</maintainer>
+  <version>1.0.0</version>
+  <description>Former AI Institute's wrappers for ROS 2, an alias for rainbows</description>
```
Comment (khughes-bdai) [inline bdai_ros2_wrappers/package.xml:6]: leftover `rainbows` 🌈

**Comment by mhidalgo-bdai** [comment]
Moving forward

**Pull request merged**